### PR TITLE
add a `make install DESTDIR=` target, update docs, and add a script to sync sources to a VM

### PR DESCRIPTION
**Pull request by rfairley**
For more details, please see the commit messages. This is based on #55, to keep documentation consistent (e.g. the mention of enabling `.path` units rather than `.service`).

This would be easiest to review going through the commits one-by-one - tried to keep individual commits small and without undoing changes in between.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,9 +6,52 @@ PACKAGE = console-login-helper-messages
...
+	@echo "(Nothing to build)"
+
+.PHONY: install
+install: all
+	(set -euo pipefail; \
+	# package-specific directories \
+	mkdir -p $(DESTDIR)$(PREFIX)/lib/$(PACKAGE)/issue.d; \
+	mkdir -p $(DESTDIR)$(PREFIX)/lib/$(PACKAGE)/motd.d; \
+	mkdir -p $(DESTDIR)$(RUNSTATEDIR)/$(PACKAGE)/issue.d; \
+	mkdir -p $(DESTDIR)$(RUNSTATEDIR)/$(PACKAGE)/motd.d; \
```
Comment (jlebon) [inline Makefile:29]: Those should be created by the `tmpfiles.d` entries, no?

File: Makefile [hunk 2]
```diff
@@ -6,9 +6,52 @@ PACKAGE = console-login-helper-messages
...
+	install -DZ -m 0644 usr/lib/tmpfiles.d/* \
+		-t $(DESTDIR)$(PREFIX)/lib/tmpfiles.d/; \
+	install -DZ -m 0644 usr/lib/udev/rules.d/* \
+		-t $(DESTDIR)$(PREFIX)/lib/udev/rules.d/; \
+	install -DZ -m 0755 usr/libexec/$(PACKAGE)/* \
+		-t $(DESTDIR)$(libexecdir)/$(PACKAGE)/; \
+	install -DZ -m 0644 usr/share/$(PACKAGE)/* \
+		-t $(DESTDIR)$(PREFIX)/share/$(PACKAGE)/; \
+	# symlinks \
+	ln -sf $(RUNSTATEDIR)/$(PACKAGE)/40_$(PACKAGE).issue \
```
Comment (jlebon) [inline Makefile:49]: We should be able to drop this one too in favour of `tmpfiles.d`.

File: Makefile (hunk 1 above)
Comment (rfairley) [reply Makefile]: These will be created by `tmpfiles.d`, yes. The reason I also created these at install time was that `rpmbuild` would give an error due to not finding runtime directories which had been listed as [owned by the package](https://src.fedoraproject.org/rpms/console-login-helper-messages/blob/8b843660d88852b06f089ffaaf8ef7f6f17d4ecf/f/console-login-helper-messages.spec#_164):

```
RPM build errors:
    Directory not found: /console-login-helper-messages/build/console-login-helper-messages/.buildroot/console-login-helper-messages-6125a7fcac00b2efda54e405b45831ef15c22a20-1.fc32.x86_64/run/console-login-helper-messages
```

I'm good with keeping these out of the Makefile and adding them downstream to the RPM specfile install section - they are only needed to satisfy RPM packaging. The guidelines seem to suggest either way is acceptable (either an upstream `Makefile` creates directories under `/run`, or the runtime directories are installed through the specfile: https://docs.fedoraproject.org/en-US/packaging-guidelines/Tmpfiles.d/#_example_spec_file).

File: Makefile (hunk 2 above)
Comment (rfairley) [reply Makefile]: Updated!

**Comment by rfairley** [comment]
Merging in - changes were made from review. Can always make adjustments later on too!

**Pull request merged**